runtime/vm/FlushProcessWriteBuffers.cpp

```diff
@@ -22,11 +22,12 @@
 
 #if defined(WIN32)
 #include <windows.h>
-#elif defined(LINUX) || defined(AIXPPC) /* WIN32 */
-#include <sys/mman.h>
-#endif /* LINUX || AIXPPC */
+#endif /* WIN32 */
 
 #include "vm_internal.h"
+#if defined(J9UNIX) || defined(AIXPPC)
```

Comment:
As a general rule, we include system headers before any openj9 ones. There's no need to keep the `elif` nesting, but the include should move up.

Reply:
This is because `J9UNIX` is defined within `j9comp.h` included by `vm_internal.h`.
If it is preferred to keep system header ahead of  openj9 ones, `#if defined(J9UNIX) || defined(AIXPPC)` can be changed to `#if defined(LINUX) || defined(AIXPPC) || defined(OSX)`.

Reply:
I think that's a bit too ugly, so let's leave it.